locales-utils/src/test/java/com/spotify/i18n/locales/utils/hierarchy/LocalesHierarchyUtilsTest.java

```diff
@@ -91,20 +91,20 @@ void getParentLocale(String childLanguageTag, String parentLanguageTag) {
   }
 
   static Stream<Arguments> getParentLocale() {
-    return Map.of(
-            "en-150", "en-001",
-            "en-GB", "en-001",
-            "en-US", "en",
-            "fr", "",
-            "ja-JP", "ja",
-            "wo-Arab", "",
-            "zh-Hans", "zh",
-            "zh-Hant", "",
-            "ht", "fr-HT",
-            "zh-Hant-MO", "zh-Hant-HK")
-        .entrySet()
-        .stream()
-        .map(e -> Arguments.arguments(e.getKey(), e.getValue()));
+    return Stream.of(
+        Arguments.of("en-150", "en-001"),
+        Arguments.of("en-GB", "en-001"),
+        Arguments.of("en-US", "en"),
+        Arguments.of("fr", ""),
+        Arguments.of("ja-JP", "ja"),
+        Arguments.of("wo-Arab", ""),
+        Arguments.of("zh-CN", "zh"),
+        Arguments.of("zh-TW", "zh-Hant"),
+        Arguments.of("zh-MO", "zh-Hant-HK"),
```

Comment:
These are the only 2 values that were added. 
- `zh-TW` parent should be `zh-Hant` (fallback logic only)
- `zh-Hant-MO` has a custom parent as per CLDR: `zh-Hant-HK`. We ensure that both the right fallback logic is used, and that the right custom parent as per CLDR is returned.